### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by egadbois**
Updated screenshots

### CODE REVIEW

File: native-oci-container-instances/README.md [hunk 1]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
 3. On a browser, visit the OCI dashboard and open the side menu to locate "Developer Services" -> "Containers & Artifacts" -> "Container Registry"
 4. Select the directory in which you stored your image (the location corresponds to the ```<tenancy-namespace>``` that you tagged the image with)
 5. Ensure that the Access type is "Public"; if it is not, click "Actions" in the top-right corner and select "Change to public"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:74]:
```suggestion
5. Ensure that the Access type is "Public"; if it is not, click **Actions** in the top-right corner and select "Change to public"
```

File: native-oci-container-instances/README.md [hunk 2]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
-<img width="888" alt="Screen Shot 2023-06-06 at 4 57 54 PM" src="https://github.com/egadbois/graalvm-demos/assets/134104678/c9ec8364-5aea-40f0-8348-79339dc6578f">
+<img width="888" alt="public repo" src="https://github.com/graalvm/graalvm-demos/assets/134104678/c57e9470-e455-401d-857d-40eb8c042591">
+
 
 Create an OCI Container Instance
 -------------------------
 1. From the main OCI dashboard, open the side menu and click on "Developer Services" -> "Containers & Artifacts" -> "Container Instances"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:80]:
```suggestion
1. From the main OCI dashboard, open the side menu and click **Developer Services** -> **Containers & Artifacts** -> **Container Instances**
```

File: native-oci-container-instances/README.md [hunk 3]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
-<img width="278" alt="Screen Shot 2023-06-06 at 4 28 20 PM" src="https://github.com/egadbois/graalvm-demos/assets/134104678/5aea839b-ab79-4eec-a0b5-74927ffde179">
+<img width="450" alt="Screen Shot 2023-06-06 at 4 28 20 PM" src="https://github.com/egadbois/graalvm-demos/assets/134104678/5aea839b-ab79-4eec-a0b5-74927ffde179">
 
 2. Click "Create container instance"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:83]:
```suggestion
2. Click **Create container instance**
```

File: native-oci-container-instances/README.md [hunk 4]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
 3. Input a name and compartment location for the instance
```
Comment (bhoran) [inline native-oci-container-instances/README.md:84]:
```suggestion
3. Enter a name and compartment location for the instance
```

File: native-oci-container-instances/README.md [hunk 5]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
-4. In the "Shape" section, adjust the sliders to your desired amounts of OCPUs and Memory (For this demo, 4 OCPUs and 128 GBs will be plenty)
-<img width="1002" alt="Screen Shot 2023-06-06 at 4 29 11 PM" src="https://github.com/egadbois/graalvm-demos/assets/134104678/d71fd5a4-e6db-401e-b205-8e814b1e9091">
+4. In the "Shape" section, adjust the sliders to your desired amounts of OCPUs and Memory (1 CPU and 1 GB of memory will be enough thanks to the power of native image!)
+![Screen Shot 2023-07-12 at 12 45 27 PM](https://github.com/egadbois/graalvm-demos/assets/134104678/393de70f-3ed4-4b62-9816-639566edb874)
+
 
 5. In the "Networking" section, you may either use a pre-existing VCN or create a new one
 6. Leave the remaining options as their default and click "Next"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:90]:
```suggestion
6. Leave the remaining options as their defaults and click **Next**
```

File: native-oci-container-instances/README.md [hunk 6]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
 7. Choose a name for the container (or leave as default) and in the "Image" section click "Select image"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:91]:
```suggestion
7. Choose a name for the container (or leave as default) and in the "Image" section click **Select image**
```

File: native-oci-container-instances/README.md [hunk 7]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
 3. On a browser, visit the OCI dashboard and open the side menu to locate "Developer Services" -> "Containers & Artifacts" -> "Container Registry"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:72]:
```suggestion
3. In the Oracle Cloud Console, open the navigation menu, click **Developer Services**. Under **Containers & Artifacts**, click **Container Registry**.
```

File: native-oci-container-instances/README.md [hunk 8]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
 4. Select the directory in which you stored your image (the location corresponds to the ```<tenancy-namespace>``` that you tagged the image with)
```
Comment (bhoran) [inline native-oci-container-instances/README.md:73]:
```suggestion
4. Select the directory in which you stored your image (the location corresponds to the ```<tenancy-namespace>``` with which you tagged the image).
```

File: native-oci-container-instances/README.md [hunk 9]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
+
 
 5. In the "Networking" section, you may either use a pre-existing VCN or create a new one
 6. Leave the remaining options as their default and click "Next"
 7. Choose a name for the container (or leave as default) and in the "Image" section click "Select image"
-<img width="987" alt="Screen Shot 2023-06-06 at 4 31 31 PM" src="https://github.com/egadbois/graalvm-demos/assets/134104678/a47b09da-838a-4674-98ed-2935ad151529">
+<img width="987" alt="Create container" src="https://github.com/graalvm/graalvm-demos/assets/134104678/092d4a12-426d-4617-b8a7-3e069d9c1550">
+
 
 8. Locate the image that you pushed in the previous section according to the ```<tenancy-namespace>``` directory that you chose
```
Comment (bhoran) [inline native-oci-container-instances/README.md:95]:
```suggestion
8. Locate the image that you pushed in the previous section according to the ```<tenancy-namespace>``` directory that you chose.
```

File: native-oci-container-instances/README.md [hunk 10]
```diff
@@ -72,29 +72,34 @@ The newly created Dockerfile will be automatically stored in the "target" direct
...
 9. Create the container and allow it a moment to intialize
-<img width="1282" alt="Screen Shot 2023-06-06 at 4 31 55 PM" src="https://github.com/egadbois/graalvm-demos/assets/134104678/3ea80a66-585d-41b3-8678-35e8f78dd869">
+<img width="1282" alt="Create container" src="https://github.com/graalvm/graalvm-demos/assets/134104678/dba8c624-1fff-4c2c-883e-0287b910ff95">
+
 
 10. A successful deployment will result in the large icon box turning green and the status "ACTIVE" displayed beneath it
```
Comment (bhoran) [inline native-oci-container-instances/README.md:100]:
```suggestion
10. A successful deployment will result in the large icon box becoming green and the status "ACTIVE" displayed beneath it.
```

File: native-oci-container-instances/README.md [hunk 11]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
 2. Open the side menu and click on "Developer Services" -> "Containers & Artifacts" -> "Container Registry"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:127]:
```suggestion
2. In the Oracle Cloud Console, open the navigation menu, click **Developer Services**. Under **Containers & Artifacts**, click **Container Registry**.
```

File: native-oci-container-instances/README.md [hunk 12]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
...
 3. Select the container image that you uploaded and under the "Actions" drop down select "Delete repository"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:128]:
```suggestion
3. Select the container image that you uploaded and from the **Actions** drop-down list select **Delete repository**.
```

File: native-oci-container-instances/README.md [hunk 13]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
...
 
-![Delete repository](https://github.com/egadbois/graalvm-demos/assets/134104678/da43004b-d017-49c4-8273-5bf0a4acc2cd)
+![Delete registry](https://github.com/graalvm/graalvm-demos/assets/134104678/b8135869-fd64-48f9-989a-83c188e8b5eb)
 
 4. On the left side of the page, select "Container Instances"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:132]:
```suggestion
4. On the left side of the page, click **Container Instances**.
```

File: native-oci-container-instances/README.md [hunk 14]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
...
 5. Select the container instance that you created and under the "More actions" drop down select "Delete"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:133]:
```suggestion
5. Select the container instance that you created and from the **More actions** drop-down list select **Delete **.
```

File: native-oci-container-instances/README.md [hunk 15]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
...
 
-![Delete instance](https://github.com/egadbois/graalvm-demos/assets/134104678/5289230a-1a19-4f71-b4bb-1669e1c6f671)
+![Delete instance](https://github.com/graalvm/graalvm-demos/assets/134104678/cb439c80-6155-47de-956f-031469741e1f)
 
 6. Open the side menu and click on "Networking" -> "Virtual cloud networks"
```
Comment (bhoran) [inline native-oci-container-instances/README.md:137]:
```suggestion
6. In the Oracle Cloud Console, open the navigation menu, click **Developer Services**and then click **Virtual cloud networks**.
```

File: native-oci-container-instances/README.md [hunk 16]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
...
 7. Select the VCN that you created for your container instance
 8. Click the red "Delete" button
```
Comment (bhoran) [inline native-oci-container-instances/README.md:139]:
```suggestion
8. Click **Delete**.
```

File: native-oci-container-instances/README.md [hunk 17]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
...
 
-![Delete VCN](https://github.com/egadbois/graalvm-demos/assets/134104678/a45cbb7e-59ec-4952-9a54-899907a9e094)
+![Delete VCN](https://github.com/graalvm/graalvm-demos/assets/134104678/34ed6396-73f1-4f2d-9ec7-409b463b295a)
 
 9. Click "Scan" - this will return a list of any lingering resources that are referencing the VCN
```
Comment (bhoran) [inline native-oci-container-instances/README.md:143]:
```suggestion
9. Click **Scan** - this returns a list of any lingering resources that reference the VCN.
```

File: native-oci-container-instances/README.md [hunk 18]
```diff
@@ -122,18 +127,18 @@ Once you have completed this demo, follow these instructions to delete the creat
...
 10. Click "Delete all" to remove the listed resources (if this fails you can select each resource individually to delete them on their own pages)
```
Comment (bhoran) [inline native-oci-container-instances/README.md:144]:
```suggestion
10. Click **Delete all** to remove the listed resources (if this fails you can select each resource individually to delete them on their own pages).
```

**Comment by olyagpl** [comment]
@bhoran, @sachin-pikle, is this PR good to go?

**Comment by sachin-pikle** [comment]
Verified and confirming that this PR is outdated and no longer needed. The main branch already has the reviewed and improved files. Closing this PR.

**Pull request closed without merging**